zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/model/AttachedProcessState.scala

```diff
@@ -0,0 +1,13 @@
+package com.coralogix.zio.k8s.client.model
+
+import com.coralogix.zio.k8s.client.K8sFailure
+import com.coralogix.zio.k8s.model.pkg.apis.meta.v1.Status
+import zio.{ Chunk, Promise, Queue }
+import zio.stream.UStream
+
+case class AttachedProcessState(
+  stdin: Option[Queue[Chunk[Byte]]],
```

Comment:
Maybe it would be nicer (at least more symmetric :) if this is a sink?

Reply:
Yep, it looks better :)